packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -1844,31 +1844,20 @@ class StringProperty extends DiagnosticsProperty<String> {
   ///
   /// The [showName], [quoted], [style], and [level] arguments must not be null.
   StringProperty(
-    String name,
-    String? value, {
-    String? description,
-    String? tooltip,
-    bool showName = true,
-    Object? defaultValue = kNoDefaultValue,
+    String super.name,
+    super.value, {
+    super.description,
+    super.tooltip,
+    super.showName,
```

Comment:
This line produces the following error:

```dart
error: The type 'dynamic' of this parameter isn't a subtype of the type 'bool' of the associated super constructor parameter. (super_formal_parameter_type_is_not_subtype_of_associated at [flutter] lib/src/foundation/diagnostics.dart:1851)
```

This error happens in a couple of places in the framework after migrating to super params. So far, I have been unable to reproduce this outside of the framework.

@pq or @johnniwinther any idea what may be going on here?

Reply:
I was able to reproduce this and filed https://github.com/dart-lang/sdk/issues/48697.